testing/workflows/components/kfctl_go_test.jsonnet

```diff
@@ -225,6 +225,7 @@ local dagTemplates = [
         "-s",
         "--use_basic_auth=" + params.useBasicAuth,
         "--use_istio=" + params.useIstio,
+        "--src_dir=" + srcDir,
```

Comment:
Specify the full path of the config file?

Reply:
done

Reply:
Did you forget to push changes?
I was expecting something like
--config=/mnt/src/.../kfctl_gcp_iap.yaml